THIRD-PARTY

```diff
@@ -1,3 +1,50 @@
+** MustacheJava; version 0.9.6 -- https://github.com/spullara/mustache.java
```

Comment:
These seem like a lot of changes for this PR. Did you re-run the following command to generate this?

```
./gradlew generateThirdPartyReport
```

Reply:
No, I simply cherry picked and merged.

Reply:
The `v1.1.x` branch used a different format for the `THIRD-PARTY` file. I'd say that you should remove the changes here and then run:

```
./gradlew generateThirdPartyReport
```

That command will update your `THIRD-PARTY` file. Commit that file and include it in the PR.